docker-entrypoint.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# exit on failures
+set -e
+set -o pipefail
+
+AcademiesDatabaseConnectionString=${AcademiesDatabaseConnectionString:?}
+
+declare -A mysqlconn
+
+for keyvaluepair in $(echo "$AcademiesDatabaseConnectionString" | sed "s/ //g; s/;/ /g")
+do
+  IFS=" " read -r -a ARR <<< "${keyvaluepair//=/ }"
+  mysqlconn[${ARR[0]}]=${ARR[1]}
+done
+
+echo "Running database migrations ..."
+until /opt/mssql-tools18/bin/sqlcmd -S "${mysqlconn[Server]}" -U "${mysqlconn[UserId]}" -P "${mysqlconn[Password]}" -d "${mysqlconn[Database]}" -C -i /app/SQL/DbMigrationScript.sql -o /app/SQL/DbMigrationScriptOutput.txt
```

Comment:
Will this loop indefinitely if the database is not available?

Reply:
It will :) ...
Currently just have it like that for testing
We can adjust it when we know how it'll behave in a live environment